package.json

```diff
@@ -1195,6 +1195,9 @@
     "testPathIgnorePatterns": [
       "<rootDir>/src/decorators"
     ],
+    "watchPathIgnorePatterns": [
+        "<rootDir>/results/unit"
+    ],
```

Comment:
This fixes jests `--watch` mode. At the end of every test run, jest reporters are executed and they write their results into `<rootDir>/results/unit` directory. Unless this directory is excluded jest think watcher thinks something has changed and rerunst tests. This results in a flickering screen with nothing running. Thank you @tjohnsonBCM for finding this config option 🎉